src/esp/gfx/CubeMap.cpp

```diff
@@ -403,6 +403,10 @@ bool CubeMap::saveTexture(TextureType type,
   if (!(converter = manager.loadAndInstantiate("AnyImageConverter"))) {
     return false;
   }
+  // set image converter flags if gfx logging is quieted
+  if (!isLevelEnabled(logging::Subsystem::gfx, logging::LoggingLevel::Debug)) {
+    converter->addFlags(Magnum::Trade::ImageConverterFlag::Quiet);
+  }
```

Comment:
While you're at it there's also a `*Flag::Verbose`, could you hook it up in a similar way? That would match the `LoggingLevel::VeryVerbose` level I think.